wlm-ui/src/MainPage/Channel/ChannelList.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { AppDispatch } from '../../store';
+import {
+    channelListActions, fetchList, postExposure, postPeriod, selectChannelList,
+} from '../../store/slices/channel/channel';
+import Channel from './Channel';
+import './ChannelList.scss';
+
+const ChannelList = () => {
+    const channelListState = useSelector(selectChannelList);
+
+    useEffect(() => {
+        localStorage.setItem('channel.channelList', JSON.stringify(channelListState.channels));
+    }, [channelListState]);
```

Comment:
Why do you save the list state in the `localStorage`?

Reply:
Unless so, whenever one refreshes the page, the current channel list would be reset, i.e., removed.
I don't remember exactly, however, I updated this to a reasonable method like `sessionStorarge` in the later issue.
So please keep this in this PR.